### PR TITLE
309 easter celebration homepage

### PR DESCRIPTION
**Pull request by cetair**

### CODE REVIEW

File: ui/src/components/homepage/Easter.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    Text,
+    Stack,
+    Link,
+    SimpleGrid
+  } from '@chakra-ui/react';
+import { AiOutlineDesktop, AiOutlineInstagram } from 'react-icons/ai';
+import { GiPartyPopper } from 'react-icons/gi';
+import { useMediaQuery } from '@chakra-ui/media-query';
+
+const HeroSection = () => {
```
Comment (albertparedandan) [inline ui/src/components/homepage/Easter.js:14]: hmm, it's a bit confusing to name this `HeroSection` and have it different from the file name and also import under a different name in MainContainer.js

File: ui/src/components/homepage/Easter.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  const [isNotSmallerScreen] = useMediaQuery('(min-width:648px)');
```
Comment (albertparedandan) [inline ui/src/components/homepage/Easter.js:15]: you don't have to change it if you don't have time, but a lot of this code is duplicated in GoodFriday.js, would be better to re-use that same exact file and just handle if it's GoodFriday mode/Easter mode

File: ui/src/components/homepage/Easter.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+  return (
+    <Flex
+    w={['100%']}
+    h='110vh'
```
Comment (albertparedandan) [inline ui/src/components/homepage/Easter.js:20]: same comment as GoodFriday PR

File: ui/src/components/MainContainer.js [hunk 4]
```diff
@@ -28,8 +28,8 @@ import ScrollToTop from './helpers/ScrollToTop';
 import AboutUsContainer from './about/AboutUsContainer';
 import OnlineSermonContainer from './sermons/OnlineSermonContainer';
 import EasterContainer from './easter';
-import GoodFridayContainer from './homepage';
-
+import GoodFridayContainer from './homepage/GoodFriday';
+import EasterCelebrationContainer from './homepage/Easter';
```
Comment (albertparedandan) [inline ui/src/components/MainContainer.js:32]: got confused by the naming

File: ui/src/components/homepage/Easter.js [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    bgImage={`url(${'/images/goodfriday/bg2.png'})`}
```
Comment (albertparedandan) [inline ui/src/components/homepage/Easter.js:21]: think this shouldnt be in a goodfriday folder?

**Review by albertparedandan — CHANGES_REQUESTED** [review]

File: ui/src/components/easter/homepage/EasterCelebrationHomeContainer.js [hunk 6]
```diff
@@ -0,0 +1,154 @@
+import React from 'react';
```
Comment (albertparedandan) [inline ui/src/components/easter/homepage/EasterCelebrationHomeContainer.js:1]: Screenshot 2022-04-11 at 22 27 07

at lower/smaller devices (iphoneSE, iphone12, iphoneXR, samsung galaxy) the fonts are a bit too big and the buttons are not immediately visible. probably have to adjust font size and vertical height on these smaller devices

**Review by albertparedandan — CHANGES_REQUESTED** [review]

File: ui/src/components/easter/homepage/EasterCelebrationHomeContainer.js [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+                bg: '#BBE5D1',
+                color: 'black',
+              }}
+              _focus={{
+                outline: 'none',
+                boxShadow: '0 0 0 0 rgba(0,0,0,0) !important',
+              }}
+              as={Link}
+              style={{ textDecoration: 'none' }}
+              href="#SocialEmbed"
```
[the remark this answers is not in the document]
Comment (ksuhartono97) [reply ui/src/components/easter/homepage/EasterCelebrationHomeContainer.js:118]: this comment is not addressed @tamarayustian @cetair

[the remark this answers is not in the document]
Comment (albertparedandan) [reply ui/src/components/easter/homepage/EasterCelebrationHomeContainer.js]: yes!!!!

**Review by albertparedandan — APPROVED** [review]

**Pull request merged**